src/wheatley.ts

```diff
@@ -331,6 +335,30 @@ export class Wheatley {
         this.setup(config).catch(this.critical_error.bind(this));
     }
 
+    private *locate_components(config: core_config) {
+        const visited = config.components?.include ? new Set<string>() : undefined;
+
+        const pw = new PathScurry(import.meta.dirname);
```

Comment:
pw?

Reply:
path walker, it's what they seem to call it. should I rename it to `path_walker`?

Reply:
I see, might be better to call it path_walker